src/packages/@ncigdc/components/analysis/availableAnalysis.js

```diff
@@ -238,9 +235,14 @@ const availableAnalysis: Array<TAnalysis> = [
     setTypes: ['case'],
     validateSets: sets =>
       ['case'].every((t: any) => Object.keys(sets[t] || {}).length === 2),
-    ResultComponent: ({ sets }) => {
-      return <CohortComparison sets={sets} />;
-    },
+    ResultComponent: props =>
```

Comment:
what are your thoughts on using `branch` instead of ternaries?

```
branch(
  props => props.id.includes('demo-'),
  renderComponent(props => (
    <Demo {...props}>
      <CohortComparison sets={props.sets} message={props.message} />
    </Demo>
  )),
)(props => <CohortComparison sets={props.sets} message={props.message} />)
```

Reply:
what's the benefit... ternaries are language level feature, no need to introduce complexity imo

Reply:
I think the main benefit is separating conditional logic from the main component. but definitely adds noise. 

ok carry on!